mage_ai/api/policies/ClusterPolicy.py

```diff
@@ -5,7 +5,9 @@
 
 
 class ClusterPolicy(BasePolicy):
-    pass
+    @property
+    def entity(self):
+        return None, None
```

Comment:
Why do we override the entity with None? Should we just use the project level permission by default? Does using None entity mean a user with only permission to one pipeline can access the resource in any projects?
Same question in other resources.

Reply:
Yeah, the reason I added the `None` entity is for resources that a user could access it with any permission to the project. There are some resources that don't really belong to any entity such as the `AutocompleteItemResource`, so for those I added the `None` entity type.